airflow/operators/python.py

```diff
@@ -356,7 +356,7 @@ def __init__(
         self,
         *,
         python_callable: Callable,
-        use_dill: bool = False,
```

Comment:
I think it might be a breaking change, and thus we should rather deprecate this parameter than remove it so without announcement. Under the hood though, we could assign the `use_cloudpickle` parameter with the provided `use_dill` value. Once this depreciation reaches EOL, we can remove it.
The same applies to the rest of the classes.